src/models/scope.ts

```diff
@@ -0,0 +1,18 @@
+export const scopes = [
+  'demandA:read',
+  'demandA:prepare',
+  'demandA:create',
+  'demandA:comment',
+  'demandB:read',
+  'demandB:prepare',
+  'demandB:create',
+  'demandB:comment',
+  'match2:read',
+  'match2:prepare',
+  'match2:propose',
+  'match2:cancel',
+  'match2:accept',
+  'match2:reject',
+] as const
```

Comment:
why do we need these scopes? This is use-case specific so shouldn't really be in the identity service